anyway/current_user_functions.py

```diff
@@ -0,0 +1,17 @@
+from flask_login import current_user
```

Comment:
We have `user_data.py` file. What is the reason to separate user handling to two files?

Reply:
When i created `user_data.py` i was thinking mostly about `dataclass` types(a pythonic struct, that holds data without functions).
The `current_user` is an object that represent the currently logged in user,
Today there 2 functions that get user data based on this object, my thought was to move all those functions(and other that will be created) to this file.

What do you think?

Reply:
I think it is a good idea (OO - putting the fields and code in same location).